sdk/src/azure/core/az_context.c

```diff
@@ -64,9 +64,9 @@ az_context_create_with_expiration(az_context const* const parent, int64_t expira
 }
 
 AZ_NODISCARD az_context az_context_create_with_value(
-    az_context const* const parent,
-    void const* const key,
-    void const* const value)
+    az_context const* parent,
```

Comment:
I think you need to reformat the code (Ctrl+K+D in VS) - this specific instance will likely become single-line.